fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -80,6 +80,8 @@
 
     public static final String PROPERTIES_INMEMORY = "in_memory";
 
+    public static final String PROPERTIES_USE_PERSISTENT_INDEX = "use_persistent_index";
```

Comment:
enable_persistent_index may be a better name.

Reply:
> enable_persistent_index may be a better name.

done, change in https://github.com/StarRocks/starrocks/pull/4174/commits/129ea923a83dc1c1b2ceba80a91d93a84201cb31